beacon-chain/blockchain/init_test.go

```diff
@@ -6,7 +6,7 @@ import (
 
 func init() {
 	// Override network name so that hardcoded genesis files are not loaded.
-	cfg := params.BeaconConfig()
-	cfg.ConfigName = "test"
-	params.OverrideBeaconConfig(cfg)
+	if err := params.Registry.SetActive(params.MainnetTestConfig()); err != nil {
+		panic(err)
+	}
```

Comment:
There are several `init_test.go` files like this. They all change `ConfigName` to prevent the genesis startup code from loading the compiled-in genesis state. This is messy in the `Registry` model because the result collides with the real mainnet and breaks the rule of no 2 configs sharing a conflicting fork schedule with under different names. So I defined this `MainnetTestConfig` value to use in these instances - a config that is almost identical to mainnet, except different fork versions, and a different name.